ui/store/actions.js

```diff
@@ -2239,10 +2239,9 @@ export function showModal(payload) {
   };
 }
 
-export function hideModal(payload) {
+export function hideModal() {
   return {
     type: actionConstants.MODAL_CLOSE,
```

Comment:
Note: there's no payload used in reducer, and there's no payload passing in the dispatcher. Hence the payload param is removed ~